From f78c31254b95d71c5d2ac90c6486820b05be6458 Mon Sep 17 00:00:00 2001
From: Evan Wang <xswang@marvell.com>
Date: Tue, 8 Aug 2017 11:11:50 +0800
Subject: [PATCH 1840/2241] fix: dts: a37x0: correct the AVS core clock

The AVS clock here should be CPU colck in order to
make AVS work in OPP framework.
The CPU clock is 16@nb_periph_clk.

Change-Id: Ieb3f3f496a4fc5cf13c4892f2706e5624efee2d8
Signed-off-by: Evan Wang <xswang@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/42660
Reviewed-by: Victor Gu <xigu@marvell.com>
Tested-by: iSoC Platform CI <ykjenk@marvell.com>
---
 arch/arm64/boot/dts/marvell/armada-37xx.dtsi | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/marvell/armada-37xx.dtsi b/arch/arm64/boot/dts/marvell/armada-37xx.dtsi
index daeb6f6..55f2312 100644
--- a/arch/arm64/boot/dts/marvell/armada-37xx.dtsi
+++ b/arch/arm64/boot/dts/marvell/armada-37xx.dtsi
@@ -154,7 +154,7 @@
 			reg_avs: avs@11500 {
 				compatible = "marvell,armada-3700-avs";
 				reg = <0x11500 0x40>;
-				clocks = <&nb_periph_clk 6>;
+				clocks = <&nb_periph_clk 16>;
 				status = "disabled";
 			};
 
-- 
2.7.4

